org-code-javabuilder/theater/src/main/java/org/code/theater/TheaterProgressPublisher.java

```diff
@@ -26,6 +27,13 @@ public TheaterProgressPublisher() {
 
   public void onPause(double seconds) {
     this.pauseTimeSeconds += seconds;
+    if (this.pauseTimeSeconds > MAX_TIME_S) {
+      String message =
```

Comment:
(something to maybe make a future task for): should we make this translatable?

Reply:
Good call. Here's the task! https://codedotorg.atlassian.net/browse/CSA-1108